### ISSUE TITLE
New feature - Develop endpoint for HR email address storage

### ISSUE THREAD

**Issue by muzammildafedar** [muzammildafedar/udayah#4]
**Description:**

To support the new feature for adding HR email addresses, we need to create a backend endpoint that will handle the data received from the Flutter application and store it in a PostgreSQL database.

**Feature Requirements:**

> 
> **Endpoint Creation:**
> 
> Develop a RESTful API endpoint (e.g., POST /api/hr-emails) to accept the following data:
> HR Email Address
> Company Name
> Website
> Database Schema:
> 
> Design a PostgreSQL table to store the HR email addresses along with the associated company names and websites.

> **Data Validation**: 
> Implement server-side validation to ensure the incoming data is correct and secure.

> **Error Handling**: 
> Provide appropriate error messages and handling for different scenarios (e.g., duplicate entries, validation errors).
> Documentation:
> 
> Document the endpoint, including request and response formats.

**Additional Notes**:
Ensure that the endpoint is secure and follows best practices for API development.
Let's discuss the database schema and implementation details.

**Comment by Rakan-Makhashin** [comment]
will the PostgreSQL DB be hosted locally with the server or using a cloud service?

**Comment by muzammildafedar** [comment]
> will the PostgreSQL DB be hosted locally with the server or using a cloud service?

We have PostgreSQL hosted on AWS. For development purposes, you can use your own local database.before starting, I would like to request that you set up the MVC structure, Express, ORM, and any other necessary components for the backend, along with proper documentation and architecture. Please create a new issue for the initial backend setup. For backend development, kindly write the code in the functions/ folder. Feel free to share any ideas you may have!

**Comment by muzammildafedar** [comment]
The reported issue has been successfully fixed. PR  #10
